package/kernel/mac80211/patches/ath9k/513-ath9k_add_pci_ids.patch

```diff
@@ -1,30 +0,0 @@
---- a/drivers/net/wireless/ath/ath9k/hw.c
-+++ b/drivers/net/wireless/ath/ath9k/hw.c
-@@ -662,6 +662,7 @@ int ath9k_hw_init(struct ath_hw *ah)
- 
- 	/* These are all the AR5008/AR9001/AR9002/AR9003 hardware family of chipsets */
- 	switch (ah->hw_version.devid) {
-+	case AR9300_DEVID_INVALID:
- 	case AR5416_DEVID_PCI:
- 	case AR5416_DEVID_PCIE:
- 	case AR5416_AR9100_DEVID:
---- a/drivers/net/wireless/ath/ath9k/hw.h
-+++ b/drivers/net/wireless/ath/ath9k/hw.h
```

Comment:
There was a discussion about this PCI ID in this patch already:
https://patchwork.ozlabs.org/project/openwrt/patch/20220116213025.1379508-1-hauke@hauke-m.de/

Reply:
That link is to a TD-W8970, which has

compatible = "pci168c,002e";

already.

Although the node is wrong: pci168c,002e  # AR9287 according to Documentation/devicetree/bindings/net/wireless/qca,ath9k.yaml

pci168c,0030 is the correct one.

Reply:
Added commit to fix this.